packages/components/psammead-radio-schedule/startTime/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Is it normal to deal with sub-components this way or should it be placed as a top level component? E.g. `@bbc/psammead-radio-schedule-starttime`

Reply:
This is the way we have done it previously e.g. in navigation https://github.com/bbc/psammead/tree/latest/packages/components/psammead-navigation/scrollable so that is why I follow this approach

Reply:
Ah cool, I was hunting around and couldn't find other examples. That's fine if it's the standard approach.